app/plugin.go

```diff
@@ -146,6 +148,11 @@ func (a *App) InitPlugins(pluginDir, webappPluginDir string) {
 	}
 	a.SetPluginsEnvironment(env)
 
+	if synchErr := a.SynchPlugins(); synchErr != nil {
```

Comment:
nit: I read `synch` as `cinch`. I'd recommend either `Synchronize` or abbreviate this to just `Sync` everywhere.

Reply:
Separately, I think it's unnecessary to declare a new `synchErr` variable. Suggest writing as:
```go
err := a.SyncPlugins();
if err != nil {
    // ...
}
```

Reply:
Is there a (MM) Go coding preference the method call outside of the if ? 
```
err := a.SyncPlugins();
if err != nil {
    // ...
}
```
over 

```
if err := a.SyncPlugins(); err != nil {
    // ...
}

Reply:
Yeah, we typically follow Go idioms here, as "defined" by the Go standard library. See, for example, https://golang.org/src/net/http/transport.go?s=16404:16465#L458.